src/plugins/gitresolver/README.md

```diff
@@ -0,0 +1,23 @@
+- infos = Information about the gitresolver plugin is in keys below
+- infos/author = Name <name@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = resolver
+- infos/recommends =
+- infos/placements = rollback getresolver setresolver commit
+- infos/status = recommended productive maintained reviewed conformant compatible coverage specific unittest shelltest tested nodep libc configurable final preview 
+- infos/metadata =
+- infos/description =
+
+## Usage ##
+
+Copy this gitresolver if you want to start a new
```

Comment:
i still not really sure how much needs to be changed until it's finished (fetch/merge/push and some other stuff), so i have no idea what the final version will look like (the api is a total mess). 
there's a high chance that i'll rewrite the conflict handling completely and some other parts so i can't really document anything yet. 

but you use it just like the normale resolver -R gitresolver /path/to/file   (even that part got changed 2 times)